reactor-core/src/test/java/reactor/core/publisher/scenarios/FluxTests.java

```diff
@@ -1136,7 +1127,7 @@ public void consistentMultithreadingWithPartition() throws InterruptedException
 
 
 		latch.await(30, TimeUnit.SECONDS);
-		assertThat("Not totally dispatched: " + latch.getCount(), latch.getCount() == 0);
+		assertThat(latch.getCount()).as("Not totally dispatched: ").isEqualTo(0L);
```

Comment:
unlike with `assertThat(String, boolean)` where String is the explanation message, `as` accepts the description of the value, so it shouldn't be inverted